### ISSUE TITLE
[BUG] [Linter] H037 false positive on a Go template comment in a conditional attribute group

### ISSUE THREAD

**Issue by silverwind** [djlint/djLint#2299]
## System Info
 - djLint Version: 1.42.1
 - Python Version: 3.14
 - template language: golang

## Issue

1.40.8 fixed H037 for mutually exclusive branches (#647, #2246), but the check stops working once a Go comment `{{/* … */}}` appears anywhere in the group. Django's `{# … #}` is unaffected.

## How To Reproduce

```hbs
<a {{if .A}}href="a"{{else}}{{/* c */}}href="b"{{end}}>x</a>
```

```
djlint --lint --profile=golang repro.tmpl
H037 1:12 Duplicate attribute found. href
```

Dropping just the comment makes it clean.

<sub>Report written by Claude Opus 5.</sub>
